upload_form.py

```diff
@@ -186,9 +186,34 @@ def uploaded_file(filename, fileformat=None):
     # column names can't have non-letters in them or javascript fails
     fix_bad_colnames(table)
 
+    column_units=[table[cln].unit for cln in table.colnames]
+    tab_metadata={'Author':'','ADS_ID':'','DOI or URL':'','Submitter':'',
+                  'IsObserved':False,'IsSimulated':False,
+                  'IsGalactic':False,'IsExtragalactic':False}
+    if len(table.meta) > 0:
+        for name, keyword in table.meta['keywords'].items():
```

Comment:
Because not all table types allow metadata, we should include a brief note in the online docs giving some examples that do allow metadata and how they'll be used.  What table type were you testing with?